cmd/provider/main.go

```diff
@@ -63,11 +64,14 @@ func main() {
 		leaderElection = app.Flag("leader-election", "Use leader election for the controller manager.").Short('l').Default("false").OverrideDefaultFromEnvar("LEADER_ELECTION").Bool()
 		leaderRenew    = app.Flag("leader-renew", "Set leader election renewal.").Short('r').Default("10s").OverrideDefaultFromEnvar("LEADER_ELECTION_RENEW").Duration()
 
-		syncInterval         = app.Flag("sync", "How often all resources will be double-checked for drift from the desired state.").Short('s').Default("1h").Duration()
-		pollInterval         = app.Flag("poll", "How often individual resources will be checked for drift from the desired state").Short('p').Default("30m").Duration()
-		reconcileConcurrency = app.Flag("reconcile-concurrency", "Set number of reconciliation loops.").Default("100").Int()
-		maxReconcileRate     = app.Flag("max-reconcile-rate", "The global maximum rate per second at which resources may checked for drift from the desired state.").Default("1000").Int()
-		reconcileTimeout     = app.Flag("reconcile-timeout", "Object reconciliation timeout").Short('t').Default("1s").Duration()
+		syncInterval          = app.Flag("sync", "How often all resources will be double-checked for drift from the desired state.").Short('s').Default("1h").Duration()
+		pollInterval          = app.Flag("poll", "How often individual resources will be checked for drift from the desired state").Short('p').Default("30m").Duration()
+		reconcileConcurrency  = app.Flag("reconcile-concurrency", "Set number of reconciliation loops.").Default("100").Int()
+		maxReconcileRate      = app.Flag("max-reconcile-rate", "The global maximum rate per second at which resources may checked for drift from the desired state.").Default("1000").Int()
+		reconcileTimeout      = app.Flag("reconcile-timeout", "Object reconciliation timeout").Short('t').Default("1s").Duration()
+		metricsExportTimeout  = app.Flag("metrics-export-timeout", "Timeout when exporting metrics").Default("2s").Duration()
+		metricsExportInterval = app.Flag("metrics-export-interval", "Interval at which metrics are exported").Default("5s").Duration()
+		metricsExportAddress  = app.Flag("metrics-export-address", "Address of otel collector").Default("opentelemetry-collector.opentelemetry:4317").String()
```

Comment:
I think these flags flags could be prefixed with `otel-` to avoid confusion.

For instance, a user might assume they are setting `manager.Options.MetricsBindAddress` by passing a value to `--metrics-export-address`